app/vep/vep_resources.py

```diff
@@ -0,0 +1,35 @@
+"""
+See the NOTICE file distributed with this work for additional information
+regarding copyright ownership.
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import logging
+from typing import Annotated
+
+from fastapi import APIRouter, responses, UploadFile, Form
+
+from core.logging import InterceptHandler
+
+logging.getLogger().handlers = [InterceptHandler()]
+
+router = APIRouter()
+
+
+
+@router.post("/submit", name="submit_vep")
```

Comment:
Didn't we agree in the [spec](https://github.com/Ensembl/ensembl-web-tools-api/commit/eac68393828b4dedc7dc42f86c36f2e02eb2221a#diff-36c51008c7a160230713fa1fc74e66dd06b23a8e73fd42d4b742cbd119451b91R21) (see also the [Confluence doc](https://www.ebi.ac.uk/seqdb/confluence/display/ENSWEB/Decisions+for+new+website+APIs#DecisionsfornewwebsiteAPIs-Generaldecisions)) that the path will be `/submissions` (`/api/tools/vep/submissions`)?

Reply:
This is API is only for testing the file upload and submission API wasn't in the scope but I'll update the name.